tests/update/slowroll_repos.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add slowroll repos in the system
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use Mojo::Base 'opensusebasetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal;
+    assert_script_run "rm /etc/zypp/repos.d/*";
+    assert_script_run "zypper -n --gpg-auto-import-keys ar --refresh http://cdn.opensuse.org/slowroll/repo/oss/ base-oss";
```

Comment:
Now those repos as discussed in a ticket (or was it jitsi) are wrong as they should come from openQA for the to be tested state, not the released repos

Reply:
a side question, why a new test module, when we could use the same or rehuse the same from leap?